_translations/id.yml

```diff
@@ -176,6 +176,7 @@ id:
     walletelectrum: "Electrum lebih fokus pada kecepatan dan kesederhanaan, dengan penggunaan sumber daya yang rendah. Electrum menggunakan server jauh untuk menangani bagian paling rumit pada sistem Bitcoin, serta memungkinkan untuk memulihkan wallet anda dengan frase rahasia."
     walletmsigna: "mSIGNA adalah jenis wallet yang canggih namun mudah digunakan, dengan fitur kecepatan dan kesederhanaan, dapat disesuaikan dengan level perusahaan, dan keamanan yang kuat. mSIGNA mendukung BIP32, transaksi dengan tanda tangan berganda, penyimpanan offline, sinkronisasi dengan banyak perangkat, dan pencadangan elektronik maupun kertas yang bersandi."
     walletbitcoinwallet: "Dompet Bitcoin mudah digunakan dan dapat diandalkan, serta aman dan cepat. Visinya adalah desentralisasi dan nol kepercayaan: tidak perlu layanan sentral untuk segala operasi yang berhubungan dengan Bitcoin. Aplikasi ini adalah pilihan yang bagus untuk pengguna awam. "
+    walletedgewallet: "Edge sudah kompatibel untuk segwit, wallet Bitcoin untuk mobile ini mempunyai level privasi tinggi, keamanan dan desentralisasi yang lebih populer untuk digunakan untuk banyak orang. Wallet Edge akan selalu ter-enkripsi, ter-back-up, meskipun server Edge sedang turun. "
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`